apps/showcase/src/components/utilities/date-picker-input-hebrew/date-picker-input-pres.context.ts

```diff
@@ -0,0 +1,23 @@
+import type {Context} from '@o3r/core';
+
+/**
+ * The ContextInput interface describes all the inputs of the component
+ */
+export interface DatePickerInputPresContextInput {
```

Comment:
This interface may extend the original context instead of redefining it no ?

Reply:
Indeed it can extend the base one. I prefer to keep it independent to have a clear separation in terms of code.